### DISCUSSION TITLE
Lock Gamemod Selection positioning

### DISCUSSION THREAD

**Discussion by KiwiKin1** [ppy/osu#22849] [Ideas] [answered]
https://user-images.githubusercontent.com/106151046/224554497-8c3a53cf-e395-4f47-ad0b-18e06c5b422f.mp4

As the video you can see, the gamemod selection component can be moveable when holding it.
I think this part getting ignored as a bug, or maybe it's an art.
I suggest developers could lock this component, since other component beside it are locked positioning (home, setting, news, changelog...).

**Comment by cdwcgt** [comment, marked as answer]
If you have a lot of custom rulesets installed, you'll know what it does

**Reply by KiwiKin1** [reply]
custom rulesets? i dont know this nor install it.
the issue i mean is from the top bar of 4 gamemod selection (osu,taiko,mania,catch ) that can be moveable

**Reply by KiwiKin1** [reply]
are you mean by custom game mod other than mania,taiko... can be installed?

**Comment by KiwiKin1** [comment]
alright, just searched for answer, previously i dont know that there were custom ruleset (custom game mod style) exist, so i taken it as a bug instead. thx for letting me know there are custom ruleset.
